### ISSUE TITLE
Radio Buttons get same ID

### ISSUE THREAD

**Issue by codebakery-ch** [MrWolfZ/ngrx-forms#63]
Radio buttons (or any Element) should not get the same IDs.

In your sample also:
```html
<div _ngcontent-c17="">
      <label _ngcontent-c17="">
        <input _ngcontent-c17="" type="radio" value="male" **id="simpleForm.sex"** name="simpleForm.sex" class="ngrx-forms-valid ngrx-forms-unsubmitted ngrx-forms-dirty ngrx-forms-touched"> Male
      </label>
      <label _ngcontent-c17="">
        <input _ngcontent-c17="" type="radio" value="female" **id="simpleForm.sex"** name="simpleForm.sex" class="ngrx-forms-valid ngrx-forms-unsubmitted ngrx-forms-dirty ngrx-forms-touched"> Female
      </label>
    </div>
```
If the label doesn't wrap the input, it's almost impossible to get a click on a label with a for="" attribute to the actual radio button.

**Comment by MrWolfZ** [comment]
While this is correct this was somewhat intentional. Sadly since there is no containing wrapper element for radio buttons I cannot easily figure out all options and give each a separate ID. There are a couple of ways I see to solve this: 1) don't provide an ID at all for `radio` inputs, 2) allow to override the `id` with a custom one, or 3) add an `idSuffix` option to the directive (to prevent you from having to manually merge the form control ID with e.g. the index of a *ngFor loop). 2) and 3) could of course both be implemented together. What do you think? Which one would you prefer?

**Comment by MrWolfZ** [comment]
Actually, having looked at the code this is indeed a bug. The way I wrote the form view adapter it should not set the `id` of the `input` elements. However, another form view adapter is accidentally applied which sets the ID. I will therefore fix this issue by not setting the `id` of `input[type=radio]` at all as I (apparently) intended from the start.

**Comment by codebakery-ch** [comment]
hm, I actually don't know what the solution would be. I think the suffix Idea is pretty neat and (as I would guess) not too hard to implement. Actually, if you don't need the id in the later functions, I would almost prefer option 1), as you would give "me more freedom"...

But since it's a bug now, I leave you to it 😄 It's not just that it "doesn't work" with non wrapping labels, it's (to me) even more of a HTML Validation thing...

**Comment by MrWolfZ** [comment]
I have just released version [3.0.0](https://github.com/MrWolfZ/ngrx-forms/releases/tag/v3.0.0) which contains a fix for this.
